ui/app/css/itcss/components/item.scss

```diff
@@ -0,0 +1,116 @@
+.Item__flex-grid {
+  width: 100%;
+  min-width: 374px;
```

Comment:
Will need some additional css to accomodate the popup view, which is 357px wide. Perhaps a media query.